wiki/Game_mode/osu!catch/en.md

```diff
@@ -74,111 +74,13 @@ The placement of in-game cursor does not matter when playing normally. If [Relax
 
-Scoring section details all the intricacies of scoring, including mathematical formula.
+- [Judgement](/wiki/Gameplay/Judgement) only provides fixed score values for each caught hit object and does not depend on hit accuracy.
+- [Combo](/wiki/Gameplay/Combo_(score_multiplier)) is a score multiplier: clearing a hit object contributes more to the total score when combo is high and vice versa. In osu!catch, combo may be [broken](/wiki/Glossary/Combobreak) by missing a large fruit or a drop, which also decreases [health](/wiki/Beatmapping/Health).
+- [Accuracy](/wiki/Gameplay/Accuracy#osu!catch) shows how many objects were caught in total. Small droplets decrease accuracy when missed, but do not break combo.
 
-### Grades
-
-| Grade | Condition |
-| :-: | :-- |
-| SS | 100.00% Accuracy |
-| S | 98.01% to 99.99% Accuracy (an 'S' rank is possible even with several misses, like in *osu!mania*). |
-| A | 94.01% to 98.00% Accuracy |
-| B | 90.01% to 94.00% Accuracy |
-| C | 85.01% to 90.00% Accuracy |
-| D | Any other accuracy under 85.00%. |
-
-A silver S or SS can be obtained by using the *[Hidden](/wiki/Game_modifier/Hidden)* and/or *[Flashlight](/wiki/Game_modifier/Flashlight)* mod.
-
-### Accuracy
-
-Accuracy is defined by how much caught over how much missed, excluding spinner's bananas.
-
-In other words: `Accuracy = Total numbers of fruits caught / Total number of fruits`
-
-| Term | Formula |
-| :-: | :-- |
-| **Total numbers of fruits caught** | Droplets + Drops + Fruits |
-| **Total numbers of fruits** | Misses + Miss droplets + Droplets + Drops + Fruits |
-
-**Exception:**
-
-- Note that bananas (or spinner fruit) are considered to be bonus or extra points obtainable and does not count towards accuracy.
-
-Note that if using the API to calculate the accuracy, number of droplets is under **count50** and number of miss droplets is under **countkatu**.
-
-### Score
-
-Scoring for osu!catch follows the same multiplier as in [osu!](/wiki/Game_mode/osu!). However, the fruit that adds up the score is different from osu!.
-
-- A regular sized fruit gives a score of 300 times the multiplier.
-- A large juice drop (slider tick) gives a score of 100.
-- The smallest drop (also called "droplet", or slider trail/path) gives a score of 10.
-- Each collected bananas (collected during spinner duration) will give a static score of 1,100 regardless of mod and combo multiplier.
-
-Combo will not be lost for missing droplets (considered as *Miss Droplet* in server leaderboard) in this mode but a drop of Accuracy and score gain from it.
-
-`Score = Hit Value + [Hit Value * ((Combo multiplier * Difficulty multiplier * Mod multiplier) / 25)]`
-
-| Term | Meaning |
-| :-: | :-- |
-| **Hit Value** | The hit circle judgement (50, 100 or 300), any slider ticks, and spinner's bonus |
-| **Combo multiplier** | (Combo before this hit - 1) or 0; whichever is higher |
-| **Difficulty multiplier** | The difficulty setting for the beatmap |
```

Comment:
the difficulty multiplier isn't explained at all unlike for the osu mode, but it's probably the same thing